pom.xml

```diff
@@ -139,7 +149,12 @@
                             org.wso2.scim2.*;version=${project.version},
                             io.scim2.swagger.client.*;version=${project.version},
                             com.squareup.okio.*,
-                            com.squareup.okhttp.*,
+                            org.apache.http.auth.*; version="${httpclient.orbit.version}",
+                            org.apache.http.client.*; version="${httpclient.orbit.version}",
+                            org.apache.http.conn.*; version="${httpclient.orbit.version}",
+                            org.apache.http.impl.client; version="${httpclient.orbit.version}",
+                            org.apache.http.*;version="${httpcore.orbit.version}",
```

Comment:
check if we can remove the parent package import

Reply:
httpcore files are included in the package `org.apache.http`. So this exports only the related content in httpcore.